src/utils/getDiscordIds.ts

```diff
@@ -0,0 +1,70 @@
+import { RDS_BASE_STAGING_API_URL } from "../constants/urls";
+import { UserBackend } from "../typeDefinitions/userBackend.types";
+import * as response from "../constants/responses";
+
+export const getDiscordIds = async (
+  userIds: string[] | string
+): Promise<string[] | string> => {
+  try {
+    const url = `${RDS_BASE_STAGING_API_URL}/users/userId`;
```

Comment:
Does the route have `users/userId` or the value of a user's id?

Reply:
If you've created users/userId then this doesn't seem like a correct route.
Adding userId/:userId seems redundant. Please look into this.

Reply:
This route already existed in our backend. You are right the naming is wrong